src/julia.h

```diff
@@ -12,6 +12,7 @@ extern "C" {
 #include "libsupport.h"
 #include <stdint.h>
 #include <string.h>
+#include <inttypes.h>
```

Comment:
Can this be under a `#ifndef _COMPILER_MICROSOFT_` ? The LLVM headers apparently define the `PR*` labels themselves for MSVC, so this gives a bunch of double-definition warnings and may not be necessary if you're only using them in files that already include LLVM headers.